collection-scripts/gather_version

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# load shared functions and data when not sourced
+if [[ -z "$DIR_NAME" ]]; then
+    CALLED=1
+    DIR_NAME=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+    source "${DIR_NAME}/common.sh"
+fi
+
+function ocp_version() {
```

Comment:
@slawqo as per the new findings, do we want to change the function name into something meaningful? (I assme `ocp_version` sends the reader to the wrong direction).

Reply:
you're right, changed